rook/scripts/wait-for-rook.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+here="$(dirname "$(readlink -f "$0")")"
+echo -n "Testing if rook is ready up to 10 times..."
```

Comment:
Shouldn't this be 5 times? since the foor loop goes from 1 -> 5

```suggestion
echo -n "Testing if rook is ready up to 5 times..."
```